db/system_keyspace.cc

```diff
@@ -285,6 +285,25 @@ schema_ptr system_keyspace::raft_config() {
     return schema;
 }
 
+schema_ptr system_keyspace::repair_history() {
+    static thread_local auto schema = [] {
+        auto id = generate_legacy_id(NAME, REPAIR_HISTORY);
+        return schema_builder(NAME, REPAIR_HISTORY, std::optional(id))
+            .with_column("table_uuid", uuid_type, column_kind::partition_key)
+            .with_column("repair_time", timestamp_type, column_kind::clustering_key)
```

Comment:
This is the repair start time, yes? add a comment for that. And let's add some slack to that (maybe half an hour) to compensate for bad clocks (the table should contain the true time).

Reply:
Yes, It is the start of the repair. We already have a propagation_delay_in_seconds option. It can compensate the clock difference.

BTW, I think we should reject a node that has bad clocks, perhaps during boot up.

Reply:
Yes, I only saw it later.

Agree nodes should test each other's clocks.

Reply:
I created: https://github.com/scylladb/scylla/issues/9585 to track the clock issue.